src/features/auth/login-form/LoginFormStyles.tsx

```diff
@@ -0,0 +1,159 @@
+import { useMediaQuery, useTheme } from "@mui/material";
+
+export const QuestionBoxStylesFunct = () =>  {
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+    const QuestionBoxStyles = {
+        marginRight: isSmallScreen ? '3px' : '15px', 
+        fontSize: isSmallScreen ? '0.9rem' : '1rem', 
+        fontFamily: 'Inter, sans-serif', 
+
+    }
+
+    return { QuestionBoxStyles };
+
+}
+
+export const FormBoxStylesFunct = () =>  {
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+    const FormBoxStyles = {
+        width: isSmallScreen ? '88%' : '350px',
+        height: '520px',
+        margin: 'auto', 
+        padding: isSmallScreen ? '18px' : '40px',
+        backgroundColor:'white',
+        display: 'flex',
+        flexDirection:'column',
+        alignItemns:'center',
+        justifyContent:'center',
+        borderRadius:'8px',
+        marginBottom:'16px',
+    };
+
+    return { FormBoxStyles };
+
+}
+
+export const FormBottomStylesFunct = () =>  {
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+    const FormBottomStyles = {
+        marginBottom:'20px', 
+        height: '50px',
+        display: isSmallScreen ? 'grid' : 'flex',
+        justifyContent:'center', 
+        aligItems: 'center',
+      }
+
+    return { FormBottomStyles };
```

Comment:
No deconstruir el retorno del hook si solo envia una cosa
```suggestion
    return FormBottomStyles;
```